core/src/mindustry/world/meta/StatValues.java

```diff
@@ -325,7 +325,10 @@ public static <T extends UnlockableContent> StatValue ammo(ObjectMap<T, BulletTy
                     bt.left().defaults().padRight(3).left();
 
                     if(type.damage > 0 && (type.collides || type.splashDamage <= 0)){
-                        if(type.continuousDamage() > 0){
+                        if(type.spawnUnit != null){
+                            BulletType b = type.spawnUnit.weapons.get(0).bullet;
```

Comment:
The unit spawned might not have a weapon at  all, so this may crash.

Reply:
It should be fixed now.